apps/storybook/src/UserProfileRecentWorks.stories.tsx

```diff
@@ -1,34 +1,44 @@
 import { UserProfileRecentWorks } from '@asap-hub/react-components';
+import { number, text } from '@storybook/addon-knobs';
+import { UserProfileDecorator } from './user-profile';
 
 export default {
   title: 'Organisms / User Profile / Recent Works',
+  decorators: [UserProfileDecorator],
 };
 
 export const Normal = () => (
   <UserProfileRecentWorks
-    orcidWorks={[
-      {
-        doi: 'https://doi.org/10.7554/elife.07083',
-        title:
-          'Recognizing the importance of new tools and resources for research',
-        type: 'WEBSITE',
-        publicationDate: {
-          year: '2015',
-          month: '3',
-        },
-        lastModifiedDate: '1478865224685',
+    orcid={text('Orcid Id', '0000-0002-9945-3696')}
+    orcidWorks={Array.from({ length: number('Number of orcid works', 2) }).map(
```

Comment:
```suggestion
    orcidWorks={Array.from({ length: number('Number of ORCID works', 2) }).map(
```
This is not a Twotter shitpost Tom! :D